### ISSUE TITLE
New commits are not being displayed in pull request

### ISSUE THREAD

**Issue by lakostin** [go-gitea/gitea#13128]
- Gitea version (or commit ref):
1.13.0+dev-683-gb725c4ee5
- Git version:
2.26.2
- Operating system:
Alpine Linux
- Database (use `[x]`):
  - [X] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant
- Log gist:

## Description
After adding new commits to the branch with open pull request new commits sometimes do not appear in this pull request.

steps to reproduce:
git checkout master
git pull
git checkout feature_branch
git merge master 
git commit -a -m "resolved merge conflict"
git push

...

## Screenshots

![Screenshot from 2020-10-13 14-54-36](https://user-images.githubusercontent.com/29179059/95857992-0fa10180-0d65-11eb-938a-728eb9671097.png)
![Screenshot from 2020-10-13 14-54-24](https://user-images.githubusercontent.com/29179059/95857993-10399800-0d65-11eb-92fa-2ed61d17c938.png)

**Comment by melmus** [comment]
I have this problem on my gitea host too.

**Comment by zeripath** [comment]
The list of commits is the result of git log pr-branch...pr-base - not from when the pr was created.

**Comment by lakostin** [comment]
> The list of commits is the result of git log pr-branch...pr-base - not from when the pr was created.

@zeripath could you explain please what you mean

**Comment by zeripath** [comment]
I meant exactly what I wrote. If you have a pr head 
`head` and pr base `base`. The commits displayed are:

```
git log head...base
```

What you appear to be asking for instead is something less clever than what `git log head...base` does probably `git log head..base`.

If you figure out what the log command of that produces the results that you want we can switch to that

**Comment by dschmidt** [comment]
I just updated to Gitea 1.16.5 from (1.12.x) and now I don't see any new commits in any branch. I.e. it seems to be reproducible 100%.

Is there any news about this? 

What I noticed is that I don't see the link to create a new PR on push anymore. Has this feature been removed or is something regarding PRs seriously broken in my setup? 

Can I provide anything to help debug this?
I did not see a warning or an error in my logfile and I'm running on log level debug.
